app/__init__.py

```diff
@@ -48,6 +48,21 @@ def create_app(config_name):
 
     csrf.init_app(application)
 
+    @csrf.error_handler
+    def csrf_handler(reason):
+        if 'user_id' not in session:
+            application.logger.info(
+                u'csrf.session_expired: Redirecting user to log in page'
+            )
+
+            return application.login_manager.unauthorized()
+
+        application.logger.info(
+            u'csrf.invalid_token: Aborting request, user_id: %s', session['user_id']
+        )
+
+        abort(400, reason)
```

Comment:
Is 400 correct? Just wondering if this will reflect in the right alerting/logging pot. Do we consider this some form of login issue or a bad request? 

Thinking 400 is probably OK, just wanted to mention it.